### PR TITLE
353 mobile refactor server tests

### PR DESCRIPTION
**Pull request by invpuppet**
- [x] Change test format to MOC
- [x] Change all mobile tests for the routes:
  - [x] Auth
  - [x] Container
  - [x] Favorite
  - [X] Item
  - [x] Opinion
  - [x] Rent
  - [x] Ticket
  - [x] User
- [x] Upgrade the coverage

From
image

to
image

### CODE REVIEW

**Review by H-chauvet — APPROVED** [review]
Goood

File: Server/routes/Mobile/user.js [hunk 1]
```diff
@@ -53,24 +53,24 @@ router.put('/password', jwtMiddleware.refreshTokenMiddleware,
 router.post('/password/reset', async (req, res) => {
   const { email } = req.body
   if (!email || email === '') {
-    return res.status(401).json({ message: 'Missing fields' })
+    return res.status(401).send('Missing fields')
```
Comment (tbellicha) [inline Server/routes/Mobile/user.js:56]: Should have a status of 400 and say that's the email is missing

**Review by tbellicha — CHANGES_REQUESTED** [review]
Good job mate !

File: Configurateur/front/test/container-creation/design_screen_test.dart [hunk 2]
```diff
@@ -81,7 +81,6 @@ void main() {
 
     await tester.tap(find.text("Suivant"));
     await tester.tap(find.text("Précédent"));
-    await tester.tap(find.text("Parcourir"));
```
Comment (Louismaestre) [inline Configurateur/front/test/container-creation/design_screen_test.dart:84]: Why did you delete this but it's still in the design screen page (line 680) ?

Comment (invpuppet) [reply Configurateur/front/test/container-creation/design_screen_test.dart:84]: to fix the test

**Review by 31Nathan — COMMENTED** [review]
LGTM

**Review by tbellicha — APPROVED** [review]
